recipes/leftbar/unread_chip/unread_chip.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <dt-chip
+    class="dt-leftbar-unread-chip"
+    :class="`dt-leftbar-unread-chip__${kind}`"
+    data-qa="dt-leftbar-unread-chip"
+    :hide-close="true"
+    v-on="$listeners"
+  >
+    <template #icon>
+      <dt-icon
+        :name="`arrow-${direction}`"
+        size="300"
+      />
+    </template>
+    <span data-qa="dt-leftbar-unread-chip__label">
+      <slot />
+    </span>
+  </dt-chip>
+</template>
+
+<script>
+import { DtChip, DtIcon } from '@';
```

Comment:
don't use '@' we had some weird circular dependency issues with that at some point, use `@/components/chip` instead. I know it's a bit less convenient

Reply:
Right, I'll change this and have that in mind in the future, thanks!